client/dashboard/profile-wizard/steps/start/style.scss

```diff
@@ -20,13 +20,42 @@
 			margin-bottom: 0;
 		}
 
+		.muriel-checkbox label.components-checkbox-control__label {
+			margin-left: $gap-large;
+		}
+
+		.muriel-checkbox input {
+			background-color: $muriel-hot-purple-600;
+			border-color: $muriel-hot-purple-600;
+		}
+
+		.components-form-toggle__track {
```

Comment:
One more small change to the toggle, it appears that it's only `18px` in the design:

<img width="690" alt="Screen Shot 2019-05-28 at 2 09 59 PM" src="https://user-images.githubusercontent.com/10561050/58454853-541d0480-8152-11e9-9ae3-78216dd8e71c.png">

And the current toggle is 24px:

<img width="512" alt="Screen Shot 2019-05-28 at 2 10 37 PM" src="https://user-images.githubusercontent.com/10561050/58454875-639c4d80-8152-11e9-8ead-5563448b6a4d.png">